docs/best-practices/version-control.md

```diff
@@ -16,9 +16,10 @@ route: /version-control
 
 ### Commits, Pull Requests and Messages
 * Commit as often as possible, but usually not when the code is broken (doesn't work)
-  * so try to arrange your changes into small ones that keep everything working
-* Commit messages should be about why the change was done (what it will change as result)
-  * Mention the Jira issue (KUVA-3, for instance) in the commit message
+  * so try to arrange your changes into small self-contained and minimal commits that keep everything working
+* Write commit messages in the following manner
+  * Start the commit message title (ie. the first line of the commit message) with the JIRA issue key followed by a space followed by a very short summary of _what_ was done
```

Comment:
I guess it can depend on the project whether an issue number is included in the title or elsewhere in the commit message (or nowhere at all). I wouldn't be too strict about that. So perhaps change this into a suggestion, mentioning that `git log --oneline` shows only the title line and if the issue number is visible there, it can help in some workflows.

Reply:
@akikoskinen Since the original version already required mentioning the issue key, I wouldn't want to remove that. I'll change the wording.

Reply:
Indeed it did. Then it's ok to leave. Just wondering, if these open source projects receive contributions from outside in the form of pull requests, then what JIRA issue numbers are they supposed to enter there... 🤔